app/src/organisms/Devices/RobotSettings/AdvancedTab/AdvancedTabSlideouts/DeviceResetSlideout.tsx

```diff
@@ -74,15 +74,13 @@ export function DeviceResetSlideout({
   )
 
   const ot2CalibrationOptions =
-    // TODO(bh, 2022-11-07): update OT-2 filter when gripper calibration reset config option available
     options != null ? options.filter(opt => opt.id.includes('Calibration')) : []
   const ot3CalibrationOptions =
     options != null
       ? options.filter(
           opt =>
             opt.id === 'pipetteOffsetCalibrations' ||
-            // TODO(bh, 2022-11-07): confirm or update when gripper calibration reset config option available
-            opt.id === 'gripperCalibration'
+            opt.id === 'gripperOffsetCalibrations'
```

Comment:
we can use calibrationOptions for both now that the server side is handling the split
```suggestion
const calibrationOptions = options != null ? options.filter(opt => opt.id.includes('Calibration')) : []```
but I am struggling a bit with making the test pass

Reply:
I just checked the current version and I don't see any test errors on this component without your suggestion.

Are you still facing any errors?

Reply:
I reverted this change. Let me do that change again and we can talk about the failing test. Thank you Koji!

Reply:
@koji I just pushed. lets wait for CI to finish :-)

Reply:
its looking for "clear deck calibration" for the Flex. I tried creating a `mockResetConfigOptions` for OT-2 and one for Flex but that didn't work. I might have done this wrong or I am missing something else

Reply:
how come we can't just have one variable for ot-2 and flex calibrations? 

`options != null ? options.filter(opt => opt.id.includes('Calibration')) : []`

Reply:
I think we can and we should. waiting for @shlokamin and @koji to decide. for now I have reverted this change